package.json

```diff
@@ -66,10 +66,12 @@
         "@testing-library/cypress": "8.0.2",
         "@testing-library/jest-dom": "5.15.1",
         "@testing-library/react": "12.1.2",
+        "@types/file-saver": "^2.0.7",
```

Comment:
We have versions: `2.0.5` and `5.3.1` for fileSaver and paparse, but different versions for the @types: 2.0.7 and 5.3.14. Can we match both versions? If there's a problem to do it at least can we use exact versions for the @types? (removing the caret prefix)